src/main/java/com/microsoft/sqlserver/jdbc/SSPIAuthentication.java

```diff
@@ -5,12 +5,193 @@
 
 package com.microsoft.sqlserver.jdbc;
 
+import java.net.IDN;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.naming.NamingException;
+
+import com.microsoft.sqlserver.jdbc.dns.DNSKerberosLocator;
+
+
 /**
  * Integrated Authentication master file. Common items for kerb and JNI auth are in this interface.
  */
 
 abstract class SSPIAuthentication {
-    abstract byte[] GenerateClientContext(byte[] pin, boolean[] done) throws SQLServerException;
+    abstract byte[] generateClientContext(byte[] pin, boolean[] done) throws SQLServerException;
+
+    abstract int releaseClientContext() throws SQLServerException;

```

Comment:
I know this code was written like this before, but do we need to keep it return an int? I don't think this method needs to return anything.